app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -1,23 +1,20 @@
 package com.rentminder
 
-import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.material.Icon
 import androidx.compose.material.Text
-import androidx.compose.material.icons.Icons
-import androidx.compose.material.icons.outlined.Home
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.compose.ui.res.painterResource
-import androidx.compose.ui.text.TextStyle
-import androidx.compose.ui.text.style.TextDecoration
 
+/**
+ * Composable function for the Rent Icon text
+ */
```

Comment:
For KDoc to be valuable, it should add description over and above what can be read by the method name. 

This is redundant with the method name, and thus, isn't adding any value.